features/admin.users.v1/constants/user-management-constants.ts

```diff
@@ -93,10 +93,6 @@ export class UserManagementConstants {
     public static readonly SCIM2_USER_SCHEMA: string = "urn:ietf:params:scim:schemas:core:2.0:User";
     public static readonly BULK_REQUEST_SCHEMA: string = "urn:ietf:params:scim:api:messages:2.0:BulkRequest";
 
-    // Schema related constants.
-    public static readonly ENTERPRISESCHEMA: string = "urn:ietf:params:scim:schemas:extension:enterprise:2.0:User";
```

Comment:
Hope this wouldn't affect the saas deployment :)

Reply:
Verified